test/functional/core/fileio_spec.lua

```diff
@@ -295,10 +295,10 @@ describe('fileio', function()
     -- use async feed_command because nvim basically hangs on the prompt
     feed_command('w')
     screen:expect([[
-      {9:WARNING: The file has been changed since}|
-      {9: reading it!!!}                          |
-      {6:Do you really want to write to it (y/n)?}|
-      ^                                        |
+      {6:WARNING: The file has been changed since}|
```

Comment:
It'd be just a personal taste, but I would actually prefer shouting, intimidating warning messages that are put in red or more discernible than simple `MoreMsg` highlights.

Reply:
Hmm, I guess we could still print the first line in WARNING colors like it did before, but the main idea behind this change is to allow `ui_attach` to handle this prompt gracefully, and splitting the message into two makes it harder to do that

Reply:
I could also look into the implementation of `do_dialog` to see if we can use different highlights for the message